src/_locales/pl-PL.json

```diff
@@ -81,6 +81,8 @@
 
   "settings-option-auto-launch": "Włącz aplikację podczas uruchamiania systemu.",
 
+  "settings-option-scroll-lyrics": "Scroll lyrics automatically while playing",

```

Comment:
I know, will remove that, but later, for when you find some more stuff to correct :laughing: